build.gradle

```diff
@@ -160,3 +161,5 @@ buildScan {
     licenseAgreementUrl = 'https://gradle.com/terms-of-service'
     licenseAgree = 'yes'
 }
+
+apply from: 'gradle/license.gradle'
```

Comment:
Other gradle scripts aren't declared together. You can find `apply from` declarations all over the file. But yes, it's of course possible.